docs/source/upgrade_notes.rst

```diff
@@ -3,10 +3,26 @@
 Upgrade Notes
 =============
 
+.. _ref-upgrade-notes-v3-5:
+
+|st2| v3.5
+----------
+
+* API will now set ``secure`` and ``samesite=lax`` cookie attributes for then auth cookie which
```

Comment:
Not sure sentence reads quite right "for then auth cookie" - should it be the not then?

Reply:
Thanks - fixed.